argilla/docs/getting_started/huggingface-spaces.md

```diff
@@ -0,0 +1,147 @@
+---
+
+# Set up in Hugging Face Spaces
+
+This guide explains how to deploy your Argilla app in a Hugging Face Space and configure it according to your preferences.
+
+> For more information about Hugging Face Spaces, check the [documentation](https://huggingface.co/docs/hub/en/spaces-overview).
+
+
+## Deploy Argilla on Spaces
+
+1. Access the [Hugging Face template](https://huggingface.co/new-space?template=argilla/argilla-template-space). The template will show the Argilla Space SDK preselected.
+2. Fill in the template. Take into account the following:
+
+    -  Enable **Persistent Storage** not to lose your data. Unless you’re using a paid Space upgrade, any Space will be shut down after 48 hours of inactivity, resulting in data loss. You can do it later as detailed in ["Enable persistent storage"](#enable-persistent-storage), but all the data previous to the set up will be lost.
+    -  Configure the **Space secrets** to manage the credentials. The default credentials and instructions for adding them later are described in ["Configure the Space secrets"](#configure-the-space-secrets).
+    -  Set up the **visibility** to make your Space `public` or `private`. To access your private Space from the Argilla SDK, you will need to [configure and provide a `HF_TOKEN`](https://huggingface.co/settings/tokens)
+
+3. Click "Create Space". The status will change to `Building` and, once it switches to `Running`, your Space will be ready. If you don't see the Argilla login UI, refresh the page.
+
+!!! note "Argilla Space direct URL"
+
+    The Argilla Space direct URL provides access to a full-screen, stable Argilla instance.
+
+    It also serves as the `api_url` for reading and writing datasets using the Argilla SDK.
+
+    This URL is constructed as follows: `https://[your-owner-name]-[your_space_name].hf.space`. For instance, if the owner of the Space is `argilla` and your Space name is `demo`, it will be `https://argilla-demo.hf.space`.
+
+
+## Enable persistent storage
+
+!!! warning
+    Activate persistent storage not to loose your data. If you are not using a paid Space upgrade, any Space will be shut down after 48 hours of inactivity. When restarted, it will undergo a factory reset, resulting in the loss of all your data.
+
+    Make sure to enable persistent storage before adding any data, as the Space will restart, removing all the previous data.
+
+To enable [persistent storage](https://huggingface.co/docs/hub/spaces-storage#persistent-storage), go to the "Settings" tab on your created Space and click on the desired plan on the "Persistent Storage" section.
+
+If you haven’t enabled persistent storage, Argilla will display a warning message by default. To prevent this warning from appearing if you don’t need persistent storage, set the environment variable `ARGILLA_SHOW_HUGGINGFACE_SPACE_PERSISTENT_STORAGE_WARNING` to `false` in "Settings" > "New variable". This will suppress the warning message.
```

Comment:
@sdiazlor we could add this env var above perhaps and also update the old docs.

Reply:
Where do you mean above? In the old docs, this env var is already available.